src/sign/index.ts

```diff
@@ -113,6 +113,7 @@ export default class Client {
     const isShutter = message?.privacy === 'shutter';
     if (!message.reason) message.reason = '';
     if (!message.app) message.app = '';
+    if (!message.metadata) message.metadata = '';
```

Comment:
How about adding a default of JSON.stringify({}); ?

Reply:
Done @bonustrack made it `{}`